be/src/pipeline/local_exchange/local_exchanger.h

```diff
@@ -123,9 +139,12 @@ class Exchanger : public ExchangerBase {
     }
 
 protected:
-    bool _enqueue_data_and_set_ready(int channel_id, LocalExchangeSinkLocalState& local_state,
-                                     BlockType&& block);
-    bool _dequeue_data(LocalExchangeSourceLocalState& local_state, BlockType& block, bool* eos);
+    void _enqueue_data_and_set_ready(int channel_id, LocalExchangeSinkLocalState& local_state,
```

Comment:
Add comment to explain in which scenario the update_total_mem_usage = true or false